crypto/src/secp256k1.rs

```diff
@@ -85,9 +85,18 @@ impl Verifier<Secp256k1Signature> for Secp256k1PublicKey {
         #[cfg(not(test))]
         let message = Message::from_hashed_data::<rust_secp256k1::hashes::sha256::Hash>(msg);
 
-        let vrfy = Secp256k1::verification_only();
-        vrfy.verify_ecdsa(&message, &signature.sig.to_standard(), &self.pubkey)
```

Comment:
this library is not safe to use since it will verify valid for both signature and its flipped version. 

i was able to reproduce against the newly added unit tests. 

we need to ensure the recovered pubkey to match the original one to be sure it is not malleated. the flipped signature would fail here.

Reply:
> i was able to reproduce against the newly added prop tests.

If you did get a proptest failure, the error message allows you to record the seed of the rng, so that this specific generated case gets ran through on further test runs.

The seed is printed with the test failure, as are instruction on which file to record it in.